File: Assignment-2/Part1-Section1-SyntaxErrors/syntax_error_corrected.js
// JavaScript: Calculate the sum of an array — CORRECTED
function calculateSum(arr) {
    let total = 0;
    for (let num of arr) {
        total += num;
    }
    return total;
}

let numbers = [1, 2, 3, 4, 5];
let result = calculateSum(numbers);
console.log("Sum in JavaScript:", result);

File: Assignment-2/Part1-Section1-SyntaxErrors/syntax_error_corrected.py
# Python: Calculate the sum of an array — CORRECTED
def calculate_sum(arr):
    total = 0
    for num in arr:
        total += num
    return total

numbers = [1, 2, 3, 4, 5]
result = calculate_sum(numbers)
print("Sum in Python:", result)
